adal/authentication_context.py

```diff
@@ -75,11 +75,17 @@ def __init__(
             read timeout) <timeouts>` tuple.
         :param enable_pii: (optional) Unless this is set to True,
             there will be no Personally Identifiable Information (PII) written in log.
+        :param verify_ssl: (optional) requests verify. Either a boolean, in which case it 
+            controls whether we verify the server's TLS certificate, or a string, in which 
+            case it must be a path to a CA bundle to use. This value is ignored if env 
+            variable ADAL_PYTHON_SSL_NO_VERIFY is used.
+        :param proxies: (optional) requests proxies. Dictionary mapping protocol to the URL 
+            of the proxy.

```

Comment:
It is true that this is the same docstring used by `requests` library. But they also provide [intensive documentation](http://docs.python-requests.org/en/master/user/advanced/#proxies) to provide the examples on what "dictionary mapping" actually looks like. Since we don't currently have those documentation effort, I would suggest we at least add one more sentence here. Either explicitly mentions an example:

> Its content will look like `proxies = {
  'http': 'http://10.10.1.10:3128',
  'https': 'http://10.10.1.10:1080',
}`

Or simply add a link to its [official documentation in requests](http://docs.python-requests.org/en/master/user/advanced/#proxies) (since we are exposing this requests behavior anyway).

Reply:
I prefer external link, scale better in the long term IMHO :)